tests/pages/nodes/node/HealthTab-cy.js

```diff
@@ -36,15 +42,15 @@ describe('Node Health Tab [0fa]', function () {
       cy.get('@filterHealth').click();
       cy.get('.dropdown').find('li').contains('Healthy').click();
       cy.get('.page-content td .text-success').should(function ($row) {
-        expect($row.length).to.equal(18);
+        expect($row.length).to.be.at.least(5);
```

Comment:
Actually, it's a good point. I re-wrote the tests to be more robust: calculating the exact numbers it should show.